poleno/utils/backends.py

```diff
@@ -0,0 +1,31 @@
+from allauth.account.auth_backends import AuthenticationBackend
```

Comment:
Pls subor premenuj na `admin_login_as.py` podla nazvu feature, kedze v djangu je kopec roznych inych typov backendov, tak nech sa to nemyli. Taktiez do toho suboru pridame este `AdminLoginAsAdminMixin` s "login as" admin view.

Nezabudni analogicky premenovat aj subor s testami.

Reply:
:ok:
popripade uz mame subor `poleno/utils/admin.py`. Tu by to bolo mozno este vhodnejsie.

Reply:
Ano, v principe by to bolo ist aj do `poleno/utils/admin.py`. Ale kludne to moze byt aj v separatnom subore, len nech je ten subor nazvany tak, ze je z toho jasne, ze aka presne admin feature to je. Ak budeme mat v buducnosti vela admin features, tak `admin.py` rozdelime na viac suborov, alebo aj na cely adresar.